wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -274,11 +262,14 @@ public void run() {
     loopCache.poll();
     m_watchdog.addEpoch("buttons.run()");
 
-    m_inRunLoop = true;
     boolean isDisabled = RobotState.isDisabled();
     // Run scheduled commands, remove finished commands.
-    for (Iterator<Command> iterator = m_scheduledCommands.iterator(); iterator.hasNext(); ) {
+    for (Iterator<Command> iterator = Set.copyOf(m_scheduledCommands).iterator();
+        iterator.hasNext(); ) {
       Command command = iterator.next();
```

Comment:
```suggestion
    for (Command command : Set.copyOf(m_scheduledCommands)) {
```
Since we don't remove via the iterator, we don't need to keep track of it separately.

Reply:
wasn't sure about deleting it but makes sense to me too so done!